src/System.Windows.Extensions/src/System/Drawing/ImageConverter.cs

```diff
@@ -63,5 +75,75 @@ public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContex
         }
 
         public override bool GetPropertiesSupported(ITypeDescriptorContext context) => true;
+
+        private unsafe Stream GetBitmapStream(byte[] rawData)
+        {
+            Debug.Assert(rawData != null, "rawData is null.");
+            try
+            {
+                fixed (byte* pByte = rawData)
+                {
+                    IntPtr addr = (IntPtr)pByte;
+
+                    if (addr == IntPtr.Zero)
+                    {
+                        return null;
+                    }
+
+                    // This will be pHeader.signature, but we try to
+                    // do this first so we avoid building the structure when we shouldn't
+                    if (rawData.Length <= sizeof(OBJECTHEADER) || Marshal.ReadInt16(addr) != 0x1c15)
```

Comment:
This means it has an ole header?

Reply:
I mean, if the first int16 in the signature is `0x1c15`

Reply:
yes I think so it means it has an object header. 
the original source code is https://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/ImageConverter.cs,181

Reply:
Ok, thanks.